### ISSUE TITLE
Generate form dynamically

### ISSUE THREAD

**Issue by lhwong** [sir-dunxalot/ember-easy-form-extensions#35]
Hi,

I am working on a form render in which fields are displayed based on data model (label, type and etc.) fetched from server. This works well when I use the form to create new record.

However, in some case I want to edit the data model. I got "TypeError: this.get(...) is undefined" error as value attribute is not set. What should I do so that the value attribute can be populated with data from 'post' data object?

```
{{!--app-name/templates/posts/edit.hbs--}}

{{#form-wrapper}}
  {{#form-controls legend='Edit post' model=model.post}}
    {{!--Your inputs here--}}
{{#each field in model.form.fields}}
          {{input-group  label=field.label type=field.inputType}}

    {{/each}}    

  {{/form-controls}}

  {{form-submission}}
{{/form-wrapper}}

```

Regards,
Nick

**Comment by sir-dunxalot** [comment]
Hey Nick,

You should be able add the `value` attribute to the `{{input-group}}` component, shouldn't you? Is there a reason you can't do that and use the binding at `field.value`? Or is `value` set in a different way?

**Comment by lhwong** [comment]
Hi Duncan,

I can use the `value=field.value` (see below), however, I want its value to be set with the value of corresponding attribute of object set to `model` of `from-controls`. Is there any way I can set binding from the `model` to `value` of `input-group` such as `Ember.Binding.from("parentView.model." + (this.get('value'))).to('value');`? 

```
 {{input-group value=field.value  label=field.label type=field.inputType inputClass="form-control"}}

```

**Comment by sir-dunxalot** [comment]
Sorry for the slow response. It's probably best to add some computed property to the input group component for your use case. Either a computed property or a binding would work, as you noted. If you haven't been able fix this then let me know.

**Comment by Truffula** [comment]
I think the Ember 2.1.0 (beta) [`{{get}}` helper](http://emberjs.com/blog/2015/08/16/ember-2-1-beta-released.html#toc_code-get-code-helper) would also be useful in this instance.

**Comment by sir-dunxalot** [comment]
Form components have been removed from this library with #48 (released as `2.0.0`) so this issue no longer pertains to this library. To use Ember EasyForm components, you can use the Ember EasyForm `1.0.0.rc-1` release, which came out recently.
